dataactcore/scripts/pull_fpds_data.py

```diff
@@ -2424,6 +2516,18 @@ def main():
         if re.match('^[A-Z\s]+$', county_code.county_name):
             county_by_name[county_code.state_code][county_name] = county_code.county_number
 
+    # get and create list of duns -> exec comp data mappings
+    exec_comp_dict = {}
+    duns_list = sess.query(DUNS).filter(DUNS.high_comp_officer1_full_na.isnot(None)).all()
```

Comment:
Same note as the FABS derivations: this could be memory intensive but currently it isn't.